src/app.ts

```diff
@@ -1,20 +1,29 @@
 import { ApplicationListenEvent } from 'https://deno.land/x/oak@v10.5.1/application.ts';
 import { Application, Router } from 'https://deno.land/x/oak@v10.5.1/mod.ts';
+import { FilterExecutor } from './exception/filter/executor.ts';
+import { GuardExecutor } from './guard/executor.ts';
 import { HookExecutor } from './hook/executor.ts';
 import { hookName } from './hook/interfaces.ts';
 
 import { Injector } from './injector/injector.ts';
 import { Logger } from './logger.ts';
 import { MetadataHelper } from './metadata/helper.ts';
 import { moduleMetadataKey, ModuleOptions } from './module/decorator.ts';
+import { HandlebarRenderer } from './renderer/handlebar.ts';
 import { DanetRouter } from './router/router.ts';
 import { Constructor } from './utils/constructor.ts';
 
 export class DanetApplication {
 	private app = new Application();
 	private injector = new Injector();
 	private hookExecutor = new HookExecutor(this.injector);
-	public danetRouter = new DanetRouter(this.injector);
+	private actualRenderer = new HandlebarRenderer();
```

Comment:
FYI `actual` means "the real/true one", you might be looking for `currentRenderer` or maybe just `renderer`.

Reply:
+1